pkg/rabbitmq/producer.go

```diff
@@ -70,11 +70,12 @@ func (p *Producer) Produce(channel Channel, payload any) {
 		publishing)
 	failOnError(err, "Failed to publish a message")
 
-	log.Printf("[%s] Sent %+v", channel, payload)
+	p.logger.Info("message produced", "channel", channel, "payload", payload)
```

Comment:
I think this is more of a debug log IMHO. I would also not log the payload as it might contain sensitive information.